gemd/json/gemd_json.py

```diff
@@ -65,7 +67,8 @@ class GEMDJson(object):
 
     _link_type = LinkByUID
 
-    def __init__(self):
+    def __init__(self, scope='auto'):
+        self._scope = scope
```

Comment:
Using the `self._var` pattern is usually accompanied with a `@property` declaration somewhere so `GemdJson(scope="foo").scope` is a thing that can be done. If you don't mind not controlling the setter/getter, you can just do `self.scope = scope`.

Reply:
v. reasonable.  I wasn't sure we wanted it user visible at all, but that's pretty paternalistic.